firstuseauthenticator/_version.py

```diff
@@ -0,0 +1,15 @@
+"""firstuseauthenticator version info"""
+
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+
+version_info = (
+    0,
+    13,
+    0,
+    'dev',  # comment-out this line for a release
+)
+__version__ = '.'.join(map(str, version_info[:3]))
```

Comment:
Will this line work when commenting out dev, or show index out of bounds? Im not sure but i think we should exclude the index selector of [:3] here. But hmmm, i saw this pattern recently in jupyterhubs main repo, and then there when i made a 1.1.0b1 release and bumped it sgter to 1.1.0b2.dev.

Reply:
The slicing takes the first 3 elements from the tuple (it doesn't index the element ```version_info[3]```), so it won't go out of bounds ([docs ref](https://docs.python.org/3.5/library/stdtypes.html#sequence-types-list-tuple-range): Note no 4 )

It looked suspicious to me too when I first saw this in [oauthenticator](https://github.com/jupyterhub/oauthenticator/blob/master/oauthenticator/_version.py), but I checked it in the interpreter and later with the docs and it works.